### ISSUE TITLE
/aac

### ISSUE THREAD

**Issue by Rerd** [konsolas/AAC-Issues#1730]
Obviously /aac is there for piracy reasons, though it is used for cheaters to find out what to set their bypasses to. A quite easy solution to this is to create a completely hidden command which only people who need to check for piracy know the command.

This will stop people from easily finding out if the server is using AAC and knowing what to set their bypasses to and it will also still allow easy checking for piracy.

I don't see a real reason why /aac should still exist but comment if I'm wrong.

**Comment by mnewt00** [comment]
You could use something like ChatControl to do say `Unknown Command. Type "/help" for help` and everyone who wants to check for piracy, you could give them perms. 

And also, people would decompile the code (against ToS don't do it) and find the command that anti-piracy people use.

tl;dr use chatcontrol and people will also find the "secret command"

**Comment by Lysandr0** [comment]
> You could use something like ChatControl to do say Unknown Command. Type "/help" for help and everyone who wants to check for piracy, you could give them perms.

As far as I'm aware, /aac itself isn't an actual command, hence why you cannot block it via standard methods.

Anyways,
I'd like to throw in my own different suggestion in this ticket since it is related to the /aac command, instead of opening a new one. (Let me know if you want me to split it into another one)

It’d be cool if AAC could send out a notification to staff members when /aac is typed by players.
(I.e.: “[player] just checked the current AAC’s version” or something among these lines)
Players who try to check the anticheat version/plugin most of the time don’t really have the best intentions… :p This is just to keep an eye out for these kind of people, especially when they already have triggered a lot of checks, thus having a high likelihood to be cheating.
(A cooldown may be needed to prevent players from intentionally spamming the staff, as well as an option to turn off this feature entirely, for those who do not wish to receive such notification)
Players who have the “aac.notify” permission should be able to bypass this check.

**Comment by konsolas** [comment]
/aac isn't going anywhere for the foreseeable future.
